### PR TITLE
Show an athletes profile-page

### PR DESCRIPTION
**Pull request by kmthorsnes**
#### Description
Changes proposed in this pull request:
* Adds page that displays a single athlete
#### What I have learned working on this feature: 
* To generate pages with Ionic
* Working with providers

#### Screenshots:
screen shot 2018-04-05 at 10 54 18

### CODE REVIEW

**Comment by holgertidemand** [comment]
A screenshot would be nice

**Review by holgertidemand — APPROVED** [review]

**Review by larathordar — APPROVED** [review]
Looks good to me

**Review by davidboll — APPROVED** [review]
The code that I did not write LGTM 👍

File: src/pages/athlete/athlete.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+<ion-content padding>
+    <ng-container *ngIf="athlete" class="container">
+        <div class="athlete">
+          <h3>{{ athlete.attributes.name }}</h3>
+          <p>{{ athlete.attributes.age }}</p>
+          <p>{{ athlete.attributes.home }}</p>
+          <p>{{ athlete.attributes.starttime }} </p>
+        </div>
+      </ng-container>
```
Comment (magnus-thor) [inline src/pages/athlete/athlete.html]: format the code properly

**Review by magnus-thor — DISMISSED** [review]
Will you be setting up any e2e(acceptance) tests?

File: src/pages/athlete/athlete.html (hunk 1 above)
Comment (davidboll) [reply src/pages/athlete/athlete.html]: Ah missed that one, we'll fix it :)

**Pull request merged**